plugin-gradle/src/main/java/com/diffplug/gradle/spotless/SpotlessTask.java

```diff
@@ -28,6 +28,7 @@
 import java.util.regex.Pattern;
 import java.util.stream.Collectors;
 
+import org.apache.tools.ant.taskdefs.condition.Os;
```

Comment:
This import doesn't seem future-proof.  Perhaps adopt the `IS_WIN` kludge from `TestFixtures`?

Reply:
This still seems not-future-proof, though I now see that you are ex-Gradle, so I'll trust your assessment of which deps `apply plugin: 'java-gradle-plugin'` will magically provide us.

Reply:
ah with the rebasing I lost this change. In general ant is part of the shipped gradle api. but I'll fix this anyhow as it makes since simpler anyhow.